akka-stream/src/main/scala/akka/stream/scaladsl/Hub.scala

```diff
@@ -431,11 +431,12 @@ private[akka] class BroadcastHub[T](bufferSize: Int) extends GraphStageWithMater
             addConsumer(consumer, startFrom)
             // in case the consumer is already stopped we need to undo registration
             implicit val ec = materializer.executionContext
-            consumer.callback.invokeWithFeedback(Initialize(startFrom)).onFailure {
+            consumer.callback.invokeWithFeedback(Initialize(startFrom)).failed.foreach {
```

Comment:
this one is probably ok since it only happens on pending registration, not sure if changing to onComplete matters here.